packages/flash-swap/test/shared/fixtures.ts

```diff
@@ -20,10 +20,12 @@ import {
 import { getHTokenName, getHTokenSymbol } from "@hifi/helpers";
 import { GodModeErc20 } from "../../typechain/GodModeErc20";
 import { GodModeHToken } from "../../typechain/GodModeHToken";
-import { GodModeUniswapV2Pair } from "../../typechain/GodModeUniswapV2Pair";
 import { HifiFlashUniswapV2 } from "../../typechain/HifiFlashUniswapV2";
 import { SimplePriceFeed } from "../../typechain/SimplePriceFeed";
 import { deployGodModeErc20 } from "./deployers";
+import { Contract, utils } from "ethers";
```

Comment:
We're never importing things from the core `ethers.js` repository. Instead we're using the bespoke package for each object imported. E.g. in this case it would be:

```ts
import { Contract } from "@ethersproject/contracts";
```

And if that package is not listed as a dev dependency yet, we list it.